Lab 1.ipynb

```diff
@@ -0,0 +1,581 @@
+   "execution_count": 8,
+   "id": "e33b8186-9647-44de-b6d0-fd0d819f392b",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0.019841094744370347"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "hyp_val = .028\n",
+    "\n",
+    "prob = 1 - stats.rayleigh.cdf(hyp_val, loc = 0, scale = 0.01)\n",
+    "prob"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f4495b1a-3fd3-445f-8a68-0cb3d3934617",
+   "metadata": {},
+   "source": [
+    "To get the corresponding Xσ, we use the ppf function from earlier on the *normal* distribution function using the *rayleigh* probability found above."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "id": "1ab8b8ad-5241-4afe-8d7d-3200ca8c5c77",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "-2.057041963792887\n"
+     ]
+    }
+   ],
+   "source": [
+    "sigma = stats.norm.ppf(prob, loc = 0, scale = 1)\n",
+    "print(sigma)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "id": "cb81d443-faf4-4c34-84f2-ff88948e4731",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0.6065306597126334\n",
+      "0.27028802073873587\n"
+     ]
+    }
+   ],
+   "source": [
+    "hyp_val = .01\n",
+    "\n",
+    "prob = 1 - stats.rayleigh.cdf(hyp_val, loc = 0, scale = 0.01)\n",
+    "print(prob)\n",
+    "\n",
+    "sigma = stats.norm.ppf(prob, loc = 0, scale = 1)\n",
+    "print(sigma)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "28dab91a-8350-42f5-b7cf-ae85813e10bc",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0.1175030974154046\n",
+      "-1.1875615473802972\n"
+     ]
+    }
+   ],
+   "source": [
+    "hyp_val = .005\n",
+    "\n",
+    "prob = stats.rayleigh.cdf(hyp_val, loc = 0, scale = 0.01)\n",
+    "print(prob)\n",
+    "\n",
+    "sigma = stats.norm.ppf(prob, loc = 0, scale = 1)\n",
+    "print(sigma)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7af17d89-c0db-4636-a522-b4db4fd6faed",
+   "metadata": {},
+   "source": [
+    "As measurements get farther from the locus of the distribution, they get less likely, although in decreasing magnitude.<br>\n",
+    "Negative signs come from the fact that opposite sides of the Gaussian are equally as likely but in opposite directions, i.e. above you can go 2σ above or below the mean, but the deviation from the mean on the below side is negative, whereas it is positive on the above side."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c8cbb57f-c344-44fd-9daf-b71dead31c4c",
+   "metadata": {},
+   "source": [
+    "# Non-continuous distributions\n",
+    "### Binomial Distribution"
+   ]
```

Comment:
What's the binomial and where it is used // how is it different from continuous distributions?